crates/topos-clock/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "topos-clock"
+version = "0.1.0"
+edition = "2021"
+
```

Comment:
I for one would appreciate a `description` entry here, along with license and author fields.

Reply:
I will create a ticket to standardize all the `Cargo.toml` in order to have those fields

Reply:
Yes please, that would be lovely. :)